### PR TITLE
Updates to README, profile, & ONBOARDING files

### PR DESCRIPTION
**Pull request by krisfreedain**
### Description
Updated files with new link to COMMUNICATIONS file, now in the community repo, along with additional information regarding how to connect with others on Slack, forum, and others.

### Issues Resolved
n/a

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: ONBOARDING.md [hunk 1]
```diff
@@ -81,8 +81,11 @@ To create issues, leave comments, or submit pull requests, you'll need a GitHub
 4. Recommended - Do (8 min) [Configure 2-factor authentication](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication)
 5. Optional - Read (5 min) If you're new to using GitHub or git, we recommend reviewing “[Using GitHub's tools and processes](https://docs.github.com/en/get-started/onboarding/getting-started-with-your-github-account#part-2-using-githubs-tools-and-processes)”
 
-### OpenSearch forum account
+### OpenSearch forum & Slack accounts
```
[the remark this answers is not in the document]
Comment (krisfreedain) [reply ONBOARDING.md]: Done. Thanks @dblock

File: ONBOARDING.md [hunk 2]
```diff
@@ -81,8 +81,11 @@ To create issues, leave comments, or submit pull requests, you'll need a GitHub
...
 
-There are multiple ways to [connect with the OpenSearch community](https://opensearch.org/connect.html), but [the forum](https://forum.opensearch.org/) is the best place to start
+There are multiple ways to [connect with the OpenSearch community](https://opensearch.org/connect.html), [the forum](https://forum.opensearch.org/) and [Slack](https://www.opensearch.org/slack.html) are the best places to start
 
 - **Do (2 min)** [create an OpenSearch forum](https://forum.opensearch.org/) account
+- **Do (2 min)** [create an OpenSearch Slack](https://www.opensearch.org/slack.html) account
+
+Refer to the ['COMMUNICATIONS'](https://github.com/opensearch-project/community/blob/main/COMMUNICATIONS.md) file for additional information
```
[the remark this answers is not in the document]
Comment (krisfreedain) [reply ONBOARDING.md]: done. thanks @dblock

File: ONBOARDING.md [hunk 3]
```diff
@@ -81,8 +81,11 @@ To create issues, leave comments, or submit pull requests, you'll need a GitHub
 4. Recommended - Do (8 min) [Configure 2-factor authentication](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication)
 5. Optional - Read (5 min) If you're new to using GitHub or git, we recommend reviewing “[Using GitHub's tools and processes](https://docs.github.com/en/get-started/onboarding/getting-started-with-your-github-account#part-2-using-githubs-tools-and-processes)”
 
-### OpenSearch forum account
+### OpenSearch forum & Slack accounts
 
-There are multiple ways to [connect with the OpenSearch community](https://opensearch.org/connect.html), but [the forum](https://forum.opensearch.org/) is the best place to start
+There are multiple ways to [connect with the OpenSearch community](https://opensearch.org/connect.html), [the forum](https://forum.opensearch.org/) and [Slack](https://www.opensearch.org/slack.html) are the best places to start
```
[the remark this answers is not in the document]
Comment (krisfreedain) [reply ONBOARDING.md]: done

**Comment by krisfreedain** [comment]
edits resolved. thanks @dblock

**Review by peternied — APPROVED** [review]

**Comment by peternied** [comment]
@krisfreedain Thanks for this update!

**Pull request merged**